### ISSUE TITLE
[doc] Document sections are partly congruent

### ISSUE THREAD

**Issue by matthiasbeyer** [matthiasbeyer/imag#230]
For example we have a section "The store" and "libstore".

We should really re-sort the chapters of the document.

I will do this, of course.

**Comment by matthiasbeyer** [comment]
As this has low prio, and is blocked as some of the open PRs are altering documentation, I will put this on hold now.

**Comment by matthiasbeyer** [comment]
This can be closed after #525 .
